node/pkg/driver/node_utils.go

```diff
@@ -239,6 +243,25 @@ func (n NodeUtils) ParseFCPorts() ([]string, error) {
 	return fcPorts, nil
 }
 
+func (n NodeUtils) IsFCExists() bool {
+	if n.IsPathExists(FCPath) {
+		return n.IsDirHasContent(FCPath)
+	}
+	return false
+}
+
+func (n NodeUtils) IsDirHasContent(path string) bool {
+	f, _ := os.Open(path)
+	defer f.Close()
+
+	_, err := f.Readdir(1)
```

Comment:
What `1` mean?
It looks like a [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29).
Can we create a constant with a descriptive name instead?

Reply:
this means that it will try to read just one file in the directory.
Do you still think I need to add it as a const?

Reply:
looks like a [well documented](https://golang.org/pkg/os/#File.Readdir) number to me

Reply:
we could use a const like `MIN_FILES_IN_NON_EMPTY_DIR`, if you think it's better

Reply:
@oriyarde there are lots of information online. We would like the code to be self-explanatory though.
Thus, I would avoid using magic numbers.